worlds/ror2/__init__.py

```diff
@@ -130,26 +159,26 @@ def create_item(self, name: str) -> Item:
         return item
 
 
-def create_events(world: MultiWorld, player: int) -> None:
-    total_locations = world.total_locations[player].value
+def create_events(multiworld: MultiWorld, player: int) -> None:
+    total_locations = multiworld.total_locations[player].value
     num_of_events = total_locations // 25
     if total_locations / 25 == num_of_events:
         num_of_events -= 1
-    world_region = world.get_region("Petrichor V", player)
+    world_region = multiworld.get_region("Petrichor V", player)
 
     for i in range(num_of_events):
         event_loc = RiskOfRainLocation(player, f"Pickup{(i + 1) * 25}", None, world_region)
         event_loc.place_locked_item(RiskOfRainItem(f"Pickup{(i + 1) * 25}", ItemClassification.progression, None, player))
-        event_loc.access_rule(lambda state, i=i: state.can_reach(f"ItemPickup{((i + 1) * 25) - 1}", player))
+        event_loc.access_rule = lambda state, i=i: state.can_reach(f"ItemPickup{((i + 1) * 25) - 1}", player)
```

Comment:
This causes a error.. AssertionError: can_reach: player is required if spot is str

Reply:
I need more info on this error because I can't reproduce it. It's doing exactly as I intended.

Reply:
All I did was run a ror2 yaml and got the error

Reply:
I wrote tests and I'm still unable to reproduce this with a 250 locations yaml
[AP_08268955892101614522_Spoiler.txt](https://github.com/ArchipelagoMW/Archipelago/files/10346215/AP_08268955892101614522_Spoiler.txt)

Reply:
I did a few tests in my yaml to see whats causing it and its total_locations: random-range-125-200.. Did that change somewhere recently?

Reply:
ok found the issue. no idea why it only came up in specific instances but it was an easy fix.